shell/platform/android/io/flutter/embedding/engine/dynamicfeatures/DynamicFeatureManager.java

```diff
@@ -46,6 +47,18 @@
    */
   public abstract void setJNI(FlutterJNI flutterJNI);
 
+  /**
+   * Sets the DynamicFeatureChannel system channel to handle the framework API to directly call
+   * methods in DynamicFeatureManager.
+   *
+   * <p>A DynamicFeatureChannel is required to handle assets-only dynamic features and manually
+   * installed dynamic features.
+   *
+   * <p>Since this class may be instantiated for injection before the FlutterEngine and System
+   * Channels are initialized, this method should be called to provide the DynamicFeatureChannel.
```

Comment:
We're also telling implementers to now take this and do stuff with it right? Describe what's expected to be sent when on this channel if you subclass.